includes/class-newspack-newsletters-editor.php

```diff
@@ -52,6 +52,7 @@ public function __construct() {
 		add_action( 'rest_post_query', [ __CLASS__, 'maybe_filter_excerpt_length' ], 10, 2 );
 		add_action( 'rest_api_init', [ __CLASS__, 'add_newspack_author_info' ] );
 		add_filter( 'the_posts', [ __CLASS__, 'maybe_reset_excerpt_length' ] );
+		add_action( 'should_load_remote_block_patterns', [ __CLASS__, 'strip_block_patterns' ] );
```

Comment:
```suggestion
		add_filter( 'should_load_remote_block_patterns', [ __CLASS__, 'strip_block_patterns' ] );
```

Reply:
Fixed in https://github.com/Automattic/newspack-newsletters/pull/731/commits/ad94af1a867c4c706c749ab33c122c5d92aecfe4